dbfit-java/core/src/main/java/dbfit/api/DbObject.java

```diff
@@ -1,12 +1,11 @@
 package dbfit.api;
 
-import dbfit.fixture.StatementExecution;
 import dbfit.util.DbParameterAccessor;
 import dbfit.util.Direction;
 
 import java.sql.SQLException;
 
 public interface DbObject {
-    public StatementExecution buildPreparedStatement(DbParameterAccessor accessors[]) throws SQLException ;
+    public DbCommand buildPreparedStatement(DbParameterAccessor accessors[]) throws SQLException;
```

Comment:
I'll update it to `buildDbCommand`, exactly as in #463.

I'll create another PR for #541, omitting the API changes too.